### DISCUSSION TITLE
Include Plotly Visualization JavaScript in the Docs

### DISCUSSION THREAD

**Discussion by Masterain98** [vuepress-theme-hope/vuepress-theme-hope#3120] [Q&A] [answered]
Hi,

I'm adding some Plotly visualization charts (HTML files) to my document. A JavaScript is embedded in the HTML file or have to load it from [https://cdn.plot.ly/plotly-latest.min.js](https://cdn.plot.ly/plotly-latest.min.js). However, after including the chart file, I failed to deploy the site, with the error `SyntaxError: Tags with side effect (<script> and <style>) are ignored in client component templates.`

I noticed the following segment may be the source of error, is there any possible solution to change it to meet Vue's requirement?
``` JavaScript
<style>.cls-1 {fill: #3f4f75;} .cls-2 {fill: #80cfbe;} .cls-3 {fill: #fff;}</style>
```

The associated commit is [here](https://github.com/DGP-Studio/Snap.Hutao.Docs/pull/38)

**Comment by Mister-Hope** [comment, marked as answer]
Use iframe for these kind of things.
